app/today/[id]/(components)/BlogDetail.tsx

```
@@ -0,0 +1,75 @@
+import HeadBanner from "@/components/HeadBanner";
+import Image from "next/image";
+import { FaHeart, FaRegCommentDots } from "react-icons/fa";
+import { BiGhost } from "react-icons/bi";
+import { Textarea } from "@/components/ui/textarea";
+import { Button } from "@/components/ui/button";
+
+interface BlogDetailProps {
+  id: string;
+}
+
+export default function BlogDetail({ id }: BlogDetailProps) {
```

ESLint

[error] 'id' is defined but never used.

(@typescript-eslint/no-unused-vars)